docs/rest_api.rst

```diff
@@ -207,10 +207,66 @@ These endpoints allow mark courses as a public.
 This means the course is available to
 all site learners regardless of their Course Access Groups membership.
 
+Public Course Marks
+~~~~~~~~~~~~~~~~~~~
+
+This endpoint returns a paginated list of JSON objects in "results".
+Each object represents a public course mark.
+Each JSON has a single property ``id`` which can be used to delete
+the course mark. The mark JSON also has a sub-object representing a
+course.
+
+
+.. code-block:: bash
+
+    GET /course_access_groups/api/v1/public-courses/
+
+    {
+      "count": 50,
+      "next": "http://mydomain.com/course_access_groups/api/v1/public-courses/?limit=20&offset=20",
+      "previous": null,
+      "results": [
+        {
+          "id": 9,
+          "course": {
+            "id": "course-v1:Red+Python+2020",
+            "name": "Introduction to Python"
+          }
+        },
+        {
+          "id": 10,
+          "course": {
+            "id": "course-v1:Blue+SQL+2020",
+            "name": "Advanced Postgres Deployments"
+          }
+        }
+      ]
+    }
+
+Adding and Deleting Public Course Marks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The public course mark endpoints lets us add, modify and delete
```

Comment:
Please change to "The public course mark endpoints let us..."

Reply:
corrected.